### ISSUE TITLE
build fails?

### ISSUE THREAD

**Issue by lexoyo** [silexlabs/unifile#150]
"too many file operations"
https://travis-ci.org/silexlabs/unifile

not sure what it is or if it is important but it looks bad on the home page ;D

**Comment by JbIPS** [comment]
Yes, I plan to isolate GitHub connector in a plugin to avoid that
